src/components/wallets/WalletBalance.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="px-5">
+    <p :class="classes.statusTitle">
+      {{ balance ? `~${mathRound(Number(balance), 8)}` : 0 }}
```

Comment:
These is a function for fomating balances. Check `src/filters/currencyAmount.js`